subliminal/providers/subtitulamos.py

```diff
@@ -0,0 +1,342 @@
+        :param int season: Season to search for.
+        :param int episode: Episode to search for.
+        :param int year: Year to search for.
+        :return: The episode URL.
+
+        """
+        logger.info('Searching episode url for %s, season %d, episode %d', series, season, episode)
+
+        # attempt first with year
+        series_response = self._query_search(f'{series} ({year})')
+        if len(series_response) == 0:
+            series_response = self._query_search(series)
+
+        if len(series_response) == 0:
+            msg = 'Series not found'
+            raise NotExists(msg)
+
+        # Provides the URL for a specific episode of the series
+        show_id = series_response[0]['show_id']
+        page_content = self._read_series(f'/shows/{show_id}')
+
+        # Select season
+        season_element = next(
+            (el for el in page_content.select('#season-choices a.choice') if str(season) in el.text), None
+        )
+        if season_element is None:
+            msg = 'Season not found'
+            raise NotExists(msg)
+
+        if 'selected' not in cast(list[str], season_element.get('class', [])):
+            page_content = self._read_series(str(season_element.get('href', '')))
+
+        # Select episode
+        episode_element = next(
+            (el for el in page_content.select('#episode-choices a.choice') if str(episode) in el.text), None
+        )
+        if episode_element is None:
+            msg = 'Episode not found'
+            raise NotExists(msg)
+
+        episode_url = str(episode_element.get('href', ''))
+        if 'selected' not in cast(list[str], episode_element.get('class', [])):
+            page_content = self._read_series(episode_url)
+
+        return page_content, episode_url
+
+    def _query_provider(
+        self, series: str | None = None, season: int | None = None, episode: int | None = None, year: int | None = None
+    ) -> list[SubtitulamosSubtitle]:
+        """Query the provider for subtitles."""
+        # get the episode page content
+        soup, episode_url = self._read_episode_page(series, season, episode, year)
+
+        # get episode title
+        title = soup.select('#episode-name h3')[0].get_text().strip().lower()
+
+        subtitles = []
+        for sub in soup.select('.download-button:not(unavailable)'):
+            # read the language
+            if (
+                sub.parent is None
+                or (lang_name_element := sub.find_previous('div', class_='language-name')) is None
+                or (version_container := sub.find_previous('div', class_='version-container')) is None
+                or not isinstance(version_container, Tag)
+                or (release_group_element := version_container.select('.version-container .text.spaced')) is None
+            ):
+                continue
+
+            language = Language.fromsubtitulamos(lang_name_element.get_text().strip())
+
+            hearing_impaired = False
```

Comment:
Do we need to force `hearing_impaired = False`, meaning we are sure they ARE NOT hearing impaired? The default, `None`, means we don't know.

Reply:
I think subtitulamos doesn't have hearing impaired subtitles, but I'm not sure, so... line removed

Reply:
ok, thanks